dds/DCPS/InstanceHandle.h

```diff
@@ -36,7 +36,7 @@ class OpenDDS_Dcps_Export InstanceHandleGenerator {
 
 private:
 #ifdef ACE_HAS_CPP11
-  std::atomic<int32_t> sequence_;
+  std::atomic<long> sequence_;
```

Comment:
For the sake of consistency, I think all these 'long' values should be InstanceHandle_t (which will be typedef'd as HANDLE_TYPE_NATIVE / long anyway, but because it shows up in the API for next, they should probably match.

Reply:
The IDL type `long` is different from C++'s `long`

Reply:
Ah, sorry, I forgot the typedef was in the IDL file. Yes, it would all get mapped to CORBA::Long anyway... but that seems like further justification for having everything be specified in terms of InstanceHandle_t (or CORBA::Long).